### PR TITLE
[Candidate Params] Update Test Plan "Other" (Redmine11238)

### PR DESCRIPTION
**Pull request by stellarxo**
Update test plan in case there is no "Other" field in the caveat_options table

### CODE REVIEW

File: modules/candidate_parameters/test/TestPlan.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	* Reason for Caveat Emptor flag
 	* Comment
 6. Change each field and click on Update and make sure it updates in the front-end and back-end
-7. If you set the Caveat Emptor flag to Yes, check that a reason must be specified. If the reason is set to other, check that an explanation must be provided.
+7. If you set the Caveat Emptor flag to Yes, check that a reason must be specified. If the reason is set to 'Other', check that an explanation must be provided. If 'Other' does not exist as an option in the caveat_options table, the field should not show up in the front-end.
```
Comment (ridz1208) [inline modules/candidate_parameters/test/TestPlan.md]: correct me if im wrong but the field still shows up even if 'Other' does not exist

Comment (stellarxo) [inline modules/candidate_parameters/test/TestPlan.md]: yup, you're right :)

Comment (stellarxo) [inline modules/candidate_parameters/test/TestPlan.md]: This should still be merged

Comment (stellarxo) [inline modules/candidate_parameters/test/TestPlan.md]: I fixed the bug

**Pull request merged**